src/main/java/org/codeforamerica/messaging/validators/MessagableValidator.java

```diff
@@ -0,0 +1,14 @@
+package org.codeforamerica.messaging.validators;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.apache.commons.lang3.StringUtils;
+import org.codeforamerica.messaging.models.Messageable;
+
+public class MessagableValidator implements ConstraintValidator<ValidMessageable, Messageable> {
```

Comment:
Minor: Inconsistent spelling of Messageable 😄 . Unfortunately, there is no one true way.